app/routers/session.py

```diff
@@ -62,14 +61,14 @@ async def list_sessions(
     cache: TRedisPlugin,
 ) -> ListSessionsResponse:
     """Get all session keys"""
-    keylist = []
+    keylist: list = []
     for key in await cache.keys(pattern=f"{IDPREFIX}*"):
-        if not isinstance(key, bytes):
+        if not isinstance(key, (str, bytes)):
             raise TypeError(
                 "Found a key that is not stored as bytes (stored as type "
                 f"{type(key)!r})."
             )
-        keylist.append(key.decode(encoding="utf-8"))
+        keylist.append(key)
```

Comment:
Either a check here should be perfomed whether `key` is a `str` or `bytes` and if it's `bytes` it should be decoded (similarly to how it was done previously) OR the `ListSessionsResponse` model should have its `keys_` field type updated to allow for the list to contain `bytes`.